src/styles/button.css

```diff
@@ -0,0 +1,83 @@
+.flat-button {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  text-align: center;
+  border-radius: 3px;
+  padding: 1em 2em;
+  box-shadow: none;
+  display: inline-block;
+}
+
+.trial-button {
+  background-color: #8dc63f;
+  margin-right: 20px;
+}
+
+.demo-button {
+  background-color: #3d4351;
+  margin-right: 20px;
+}
+
+@media only screen and (max-width: 720px) {
+  .flat-button {
+    width: 300px;
+    margin: auto;
+    display: flex;
```

Comment:
As an example, this rule is repeated in the media query for `max-width: 485px`.  These repeated styles can be refactored into a style for the class selector `.flat-button`, outside of any media query:
```css
.flat-button {
  display: flex;
  // etc.
}
```
Now, the repeated rules can be removed from the more specific versions of `.flat-button` in the media query...